tasks/install.src.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: collectd-install | Install dependencies
+  apt: name={{item}} force=yes
+  with_items: [build-essential, libcurl4-openssl-dev, libpcap-dev, libyajl-dev]
```

Comment:
This will create a warning (I know this is just a copy), but 
```
apt: 
    name: [build-essential, libcurl4-openssl-dev, libpcap-dev, libyajl-dev]
    force: yes
```
would do the same and will prevent the warning

Reply:
Not gonna touch it, I just renamed it plus this file is not gonna be included ever in our infra, we either install from PPA with old 16.04 or by the universe repo in 18.04